folding-schemes/Cargo.toml

```diff
@@ -58,6 +82,7 @@ getrandom = { version = "0.2", features = ["js"] }
 default = ["ark-circom/default", "parallel"]
 parallel = []
 wasm = ["ark-circom/wasm"]
+browser = ["wasm", "byteorder"]
```

Comment:
not sure I follow, is there some case where the `wasm` flag is used without the `browser` flag? I'm not able to find it other than with the `browser` flag, not sure what I'm missing

Reply:
Yes, when you want to generate Circom witness without the browser. SO exactly what we were doing now.

You can't generate witness for Circom withour browser otherwise if we don't have the wasm feature.

Reply:
Resolved in person. Changes updated in 7ec21d6